src/components/ShareStructure/EditShareStructure.vue

```diff
@@ -0,0 +1,490 @@
+                      <v-row><v-col cols="6">
+                      <v-text-field
+                        filled
+                        label="Maximum Number of Shares"
+                        id="txt-max-shares"
+                        v-model="shareStructure.maxNumberOfShares"
+                        persistent-hint
+                        :hint="'Enter the maximum number of shares in the ' + shareStructure.type.toLowerCase()"
+                        :rules="getMaximumShareRule()"
+                        :disabled="hasNoMaximumShares"/>
+                    </v-col></v-row>
+                    </template>
+                  </v-radio>
+                  <v-radio :value="true" label="No maximum" id="lbl-no-maximum" v-if="isNoMaxSharesVisible"/>
+                </v-radio-group>
+
+                <v-divider class="separator" />
+
+                <v-radio-group
+                  v-model="hasNoParValue"
+                  column
+                  class="radio-group"
+                  @change="changeParValueFlag()" v-show="isClass">
+                  <v-radio :value="false" id="radio-par-value">
+                    <template v-slot:label>
+                      <v-row>
+                        <v-col cols="6">
+                          <v-text-field
+                            filled
+                            label="Par Value"
+                            id="class-par-value"
+                            v-model="shareStructure.parValue"
+                            :rules="getParValueRule()"
+                            hint="Enter the initial value of each share"
+                            persistent-hint/>
+                        </v-col>
+                        <v-col cols="6">
+                          <v-select
+                            :items="getCurrencyList()"
+                            filled
+                            label="Currency"
+                            v-model="shareStructure.currency"
+                            :rules="getCurrencyRule()"
+                            item-text="`${data.item.name}, ${data.item.code}`"
+                            item-value="code"
+                            id='class-currency'>
+                            <template slot="selection" slot-scope="data">
+                               {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                            <template slot="item" slot-scope="data">
+                              {{ data.item.name }} ({{ data.item.code }})
+                            </template>
+                          </v-select>
+                        </v-col>
+                      </v-row>
+                    </template>
+                  </v-radio>
+                  <v-radio :value="true" label="No par value" id="radio-no-par"/>
+                </v-radio-group>
+
+                <div v-show="isSeries">
+                    <v-row v-if="shareStructure.hasParValue">
```

Comment:
Change indent width to "2 spaces" not 4. (It does this to me sometimes too.)

Reply:
thanks! Fixed in next commit